src/serve/database-sqlite.ts

```diff
@@ -18,7 +18,8 @@ export default class SqliteBackend extends DatabaseBackend {
   constructor (options: { filepath?: string } = {}) {
     super()
     const { filepath } = options
-    const resolvedPath = resolve(filepath!)
+    if (!filepath) return
```

Comment:
Should we output a `console.warn` or `console.error` here?

Reply:
No, IMHO anyhow. This is just using the default values, which allows one to use a backend without configuration.